app/domain/errors.rb

```diff
@@ -343,6 +343,21 @@ module AuthnAzure
         code: "CONJ00054E"
       )
     end
+
+    module AuthnGcp
+
+      InvalidAudience = ::Util::TrackableErrorClass.new(
+        msg:  "'audience' token claim {0} is invalid. The format should be " \
```

Comment:
malformed?

Reply:
@shulifink what do you think?

Reply:
I don't understand it 
what does it mean?